src/ptz-action-source.c

```diff
@@ -89,28 +90,64 @@ static void ptz_action_source_activate(void *data)
 		ptz_action_source_do_action(context);
 }
 
+static bool is_ptz_in_preview(struct ptz_action_source_data *context)
+{
+	obs_source_t *source = obs_frontend_get_current_preview_scene();
+	if (source) {
+		bool is_preview =
+			ptz_scene_is_source_active(source, context->src);
+		obs_source_release(source);
+		return is_preview;
+	}
+	return false;
+}
```

Comment:
Nitpick: I prefer the style of bailing on errors early rather than nesting if()s on the good case

e.g.
```
obs_source_t *source = obs...();
if (!source)
    return false;
...
return is_preview;
```

Reply:
hehe... just followed the original code convention. I don't like indentation levels too...